### ISSUE TITLE
[Insight] The response should be redirected after posting data to an action - in src/…/StatistiquesController.php, line 14

### ISSUE THREAD

**Issue by artandor** [artandor/diabetehelper#7]
I'm not sure about the method i should use. This POST is just used to submit a date to get a different graph from here : [Link](http://nmylle.eu/en/stats/)
Maybe the right method is GET, must check it or ask @vasilvestre

**[in src/DiabeteHelperBundle/Controller/StatistiquesController.php, line 14](https://github.com/artandor/diabetehelper/blob/09d1f215b45c03e11903c7d9543db5733672f587/src/DiabeteHelperBundle/Controller/StatistiquesController.php#L14)**

> The `indexAction()` method allows POST requests, but does not redirect the user. It is a common best practice to call a redirect after a POST action.

```php
use Symfony\Component\Translation\MessageSelector;
use Symfony\Component\Translation\Translator;
use Symfony\Component\HttpFoundation\Request;

class StatistiquesController extends Controller {
  public function indexAction(Request $request) {
    $renderParams = array();
    $form = $this->createForm(
      'DiabeteHelperBundle\Form\Type\StatistiquesDatesType'
    );
    $form->handleRequest($request);
```

---------------------

_Posted from [SensioLabsInsight](https://insight.sensiolabs.com/projects/ddfa8ed0-2ecc-4355-933c-6884d1599dc0/analyses/11?status=violations)_

**Comment by vasilvestre** [comment]
Definitly GET method.
You "get" stats for a period, you don't create them.

More explications :
You POST the form, but it's a GET in back. Url should be about the ressource you deal with, the content, not a simple filter.

**Comment by artandor** [comment]
Yeah the post is just a way for user to chose what to GET as ressource ; got it thanks ;)
